bertserini/reader/bert_reader.py

```diff
@@ -0,0 +1,114 @@
+from typing import List
+
+from transformers import AutoTokenizer, AutoModelForQuestionAnswering, squad_convert_examples_to_features
+from torch.utils.data import DataLoader, SequentialSampler
+import torch
+from transformers.data.processors.squad import SquadResult
+
+from bertserini.reader.base import Reader, Question, Context, Answer
+
+__all__ = ['BERT']
+
+from bertserini.train.run_squad import to_list
+
+from bertserini.utils.utils_squad import SquadExample, compute_predictions_logits
+
+
+def craft_squad_examples(question: Question, contexts: List[Context]) -> List[SquadExample]:
+    examples = []
+    for idx, ctx in enumerate(contexts):
+        examples.append(
+            SquadExample(
+                qas_id=idx,
+                question_text=question.text,
+                context_text=ctx.text,
+                answer_text=None,
+                start_position_character=None,
+                title="",
+                is_impossible=False,
+                answers=[],
+                language=ctx.language
+            )
+        )
+    return examples
+
+
+class BERT(Reader):
+    def __init__(self, model_name: str, tokenizer_name: str = None):
+        if tokenizer_name is None:
+            tokenizer_name = model_name
+        self.device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+        self.model = AutoModelForQuestionAnswering.from_pretrained(model_name).to(self.device).eval()
+        self.tokenizer = AutoTokenizer.from_pretrained(tokenizer_name, do_lower_case=True)
+
+    def predict(self, question: Question, contexts: List[Context]) -> List[Answer]:
+        examples = craft_squad_examples(question, contexts)
+
+        features, dataset = squad_convert_examples_to_features(
+            examples=examples,
+            tokenizer=self.tokenizer,
+            max_seq_length=384,
```

Comment:
I didn't put this in func parameters since this kind of variable doesn't need many changes when people using it as a package but make things look heavy.
how about create a file `setting.py` to config this kind of things? i.e. manage in a class BERTReaderSetting

Reply:
What about pass in something like bert_args which stores all the parameters?

Reply:
Yeah, i think `berg_args` is a good idea for here. Thanks!